tests/backend/test_typing.py

```diff
@@ -77,6 +146,22 @@ def test_shorthands(typing_type, element):
     assert typing_type == unparse(normalize(element))
 
 
+@pytest.mark.parametrize("typing_type, element", callables)
+def test_callables(typing_type, element):
+    assert typing_type == unparse(normalize(element))
+
+
+@pytest.mark.parametrize("typing_type, element", callable_signatures)
+def test_callable_signatures(typing_type, element):
+    assert typing_type == unparse(normalize(element))
+
+
+@pytest.mark.parametrize("element", protocol_signatures)
```

Comment:
Do you want to solve the Protocol-fallback within this PR, or push it like this?

Reply:
I'd prefer to keep them separate. Protocols/Declarations may need some restructuring of normalize/unparse that aren't strongly tied to callables.